app/assets/stylesheets/modules/_articles.styl

```diff
@@ -0,0 +1,20 @@
+.wrap-filters
+  margin-top: -50px
+  float right
+  b
+    margin-right: 10px
+  *
+    display: inline-block
+  select
+    font-size: 12px;
+    font-weight: 600;
+
+.article-sort
+  width: 60px
+  .sorts
+    background: url(../images/sort.svg) no-repeat 48% #fff;
+    -webkit-appearance: none;
+    text-indent: -200px;
+
+.tooltip-trigger
```

Comment:
What do you want it to be? I think it is better to be throughout because the tooltips don't need to be that wide. If you just want it to be for articles tab, that's fine too. Your call.